node/core/dispute-coordinator/src/real/ordering/mod.rs

```diff
@@ -119,6 +126,11 @@ impl CandidateComparator {
 }
 
 impl OrderingProvider {
+	/// Limits the number of ancestors received for a single request.
+	pub(crate) const ANCESTRY_STEP: usize = 10;
```

Comment:
`STEP` conveys a different meaning from what the doc comment states, `SEGMENT_SIZE` or `PROCESSING_CHUNK_SIZE`

Reply:
If we are processing all of them, it's not a step since we are not skipping any. If that's there, it's something to be changed as well.